src/main/java/uk/gov/cshr/controller/reset/ResetController.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.cshr.controller.reset;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import uk.gov.cshr.domain.Identity;
+import uk.gov.cshr.domain.Reset;
+import uk.gov.cshr.repository.IdentityRepository;
+import uk.gov.cshr.repository.ResetRepository;
+import uk.gov.cshr.service.ResetService;
+import uk.gov.service.notify.NotificationClientException;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("/reset")
+public class ResetController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResetController.class);
+
+    @Autowired
+    private ResetService resetService;
+
+    @Autowired
+    private ResetRepository resetRepository;
+
+    @Autowired
+    private IdentityRepository identityRepository;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    @Autowired
+    private ResetFormValidator resetFormValidator;
+
+    @GetMapping
+    public String reset() {
+        return "user-reset";
```

Comment:
STRUCTURE - I think it is about time we have created folders for the templates. Given we don't have `user` concept also, maybe rename? https://lpg-docs.cloudapps.digital/IdentityService/